source/extensions/common/tap/tap_config_base.cc

```diff
@@ -13,22 +13,31 @@ namespace Extensions {
 namespace Common {
 namespace Tap {
 
-void Utility::addBufferToProtoBytes(envoy::data::tap::v2alpha::Body& output_bytes,
-                                    uint32_t max_buffered_bytes, const Buffer::Instance& data) {
+bool Utility::addBufferToProtoBytes(envoy::data::tap::v2alpha::Body& output_bytes,
```

Comment:
Nit: this is called `output_body` in the declaration but `output_bytes` here, prefer consistency.